modules/control/common/trajectory_analyzer.h

```diff
@@ -25,13 +25,16 @@
 
 #include "modules/planning/proto/planning.pb.h"
 
+#include "modules/common/math/box2d.h"
+#include "modules/common/math/vec2d.h"
 #include "modules/common/proto/pnc_point.pb.h"
 #include "modules/common/vehicle_state/vehicle_state_provider.h"
 
 /**
  * @namespace apollo::control
  * @brief apollo::control
  */
+namespace math = apollo::common::math;
```

Comment:
Don't create namespace alias, especially in header files. It will pollute whole binary.